Pegasus/sources/Math.cpp

```diff
@@ -194,6 +196,22 @@ void JacobiEigenvalue::Calculate()
     m_eigenvectors = S;
 }
 
+glm::dvec3 pegasus::math::CalculateCentroid(std::initializer_list<glm::dvec3> vectors)
```

Comment:
question same as above

Reply:
This function duplicates `pegasus::math::CalculateExpectedValue`

Reply:
@ilia-glushchenko actually, yeah, it does. When I started this branch, it wasn't the case, but now `CalculateExpectedValue` is just what I need.

Btw, shouldn't we rename `CalculateExpectedValue` to something more appropriate? Like `CalculateCentroid`, for instance.

Reply:
@yanenkoa Why? It does what it says it does, calculates expected value. 
https://en.wikipedia.org/wiki/Expected_value

Reply:
Well, in literature I've read "expected value" usually refers to a theoretical value, and this calculates sample estimation of this theoretical value, which is usually referred to as mean value. But whatever, it doesn't matter.

Reply:
@yanenkoa I've looked into it. Okay. You can make it `CalculateMeanValue` than.

Reply:
@ilia-glushchenko cool.